scripts/globals/wsquest.lua

```diff
@@ -0,0 +1,634 @@
+require('scripts/globals/keyitems');
+require('scripts/globals/quests');
+require('scripts/globals/status');
+
+dsp = dsp or {};
+dsp.wsquest =
+{
+    --------------------------------------
+    -- Quest: The Walls of Your Mind
+    --   NPC: Oggbi (Port Bastok)
+    --    WS: Asuran Fists (Hand-to-Hand)
+    --------------------------------------
+    ['theWallsOfYourMind'] =
+    {
+        ['questName'] = 'The Walls Of Your Mind',
+        ['wsName'] = 'Asuran Fists',
+        ['wsId'] = 9,
+        ['wsUnlockId'] = 1,
+        ['skillId'] = dsp.skill.HAND_TO_HAND,
+        ['minSkill'] = 250,
+        ['trialWeaponName'] = 'Knuckles of Trials',
+        ['trialWeaponId'] = 17507,
+        ['logId'] = BASTOK,
+        ['questId'] = THE_WALLS_OF_YOUR_MIND,
+        ['fameRegion'] = BASTOK,
+        ['wsnmName'] = 'Bodach',
+        ['wsnmId'] = 17461479,
```

Comment:
I really don't like the idea of having _another_ spot we have to look for entity ids in the event they shift, however unlikely these zones are to ever see one again.

Perhaps just put them in the respective zones MobIDs.lua and in the QM script just pass it along, 

eg:
```cpp
handleQmTrigger(player, dsp.wsquest.theWallsOfYourMind, BODACH)
```

Reply:
I failed to notice this earlier, good thing I left it longer for more yes. That needs moved into MobIDs per zone and then getWsnmVariableName(quest) can we  `require` the correct zone's ID's dynamically by using concatenation
```lua
require("scripts/zones/" .. player:getZoneName() .. "/MobIDs";
```

Reply:
if it has to be there instead of the scripts in each zone I mean